docs/content/en/tasks/backup-restore/_index.md

```diff
@@ -154,58 +149,199 @@ status:
 ```
 
 All pods having completed the backup will be in the `finished` list.
+At the end of the backup operation, a `MedusaBackup` custom resource will be created with the same name as the `MedusaBackupJob` object. It materializes the backup locally on the Kubernetes cluster.
+For a restore to be possible, a `MedusaBackup` object should exist.
+
+
+## Creating a Backup Schedule
+
+K8ssandra-operator v1.2 introduced a new `MedusaBackupSchedule` CRD to manage backup schedules using a [cron expression](https://crontab.guru/):  
+
+```yaml
+apiVersion: medusa.k8ssandra.io/v1alpha1
+kind: MedusaBackupSchedule
+metadata:
+  name: medusa-backup-schedule
+  namespace: k8ssandra-operator
+spec:
+  backupSpec:
+    backupType: differential
+    cassandraDatacenter: dc1
+  cronSchedule: 30 1 * * *
+  disabled: false
+```
+
+The above definition would trigger a differential backup of `dc1` every day at 1:30 AM. The status of the backup schedule will be updated with the last execution and next execution times:
+
+```yaml
+...
+status:
+  lastExecution: "2022-07-26T01:30:00Z"
+  nextSchedule: "2022-07-27T01:30:00Z"
+...
+```
+
+The `MedusaBackupJob` and `MedusaBackup` objects will be created with the name of the `MedusaBackupSchedule` object as prefix and a timestamp as suffix, for example: `medusa-backup-schedule-1658626200`.
 
 ## Restoring a Backup
 
 To restore an existing backup for a Cassandra datacenter, create the following custom resource in the namespace where K8ssandra was deployed. Example:
 
 ```yaml
 apiVersion: medusa.k8ssandra.io/v1alpha1
-kind: CassandraRestore
+kind: MedusaRestoreJob
 metadata:
   name: restore-backup1
   namespace: k8ssandra-operator
 spec:
-  cassandraDatacenter: 
-    name: dc1
-    clusterName: demo
+  cassandraDatacenter: dc1
   backup: medusa-backup1
-  inPlace: true
-  shutdown: true
 ```
 
-The `spec.backup` value should match the CassandraBackup `spec.name` value.  
-Once the K8ssandra Operator detects on the `CassandraRestore` object creation, it will control the shutdown of all Cassandra pods, and the `medusa-restore` container will perform the actual data restore upon pod restart.
+The `spec.backup` value should match the `MedusaBackup` `metadata.name` value.  
+Once the K8ssandra Operator detects on the `MedusaRestoreJob` object creation, it will orchestrate the shutdown of all Cassandra pods, and the `medusa-restore` container will perform the actual data restore upon pod restart.
 
 ### Checking Restore Completion
 
-To monitor the restore completion, check if the `finishTime` value isn't empty in the `CassandraRestore` object status. Example:
+To monitor the restore completion, check if the `finishTime` value isn't empty in the `MedusaRestoreJob` object status. Example:
 
 ```sh
 % kubectl get cassandrarestore/restore-backup1 -o yaml
 
```

Comment:
The **Restoring a Backup** section says:

>To restore an existing backup for a Cassandra datacenter, create the following custom resource in the namespace where K8ssandra was deployed.

Can you please change this to make it clear it needs to be deployed in the same namespace and in the same k8s cluster in which the CassandraDatacenter exists.

Reply:
👍